app/src/main/res/layout/otp_fragment.xml

```diff
@@ -8,44 +8,71 @@
 
 
     <TextView
```

Comment:
@raghav32 Please fix layout as per design. Currently your design is coming in the horizontal central, as per design it should be left aligned